suites/quincy/rados/tier-2_rados_test_parameters.yaml

```diff
@@ -0,0 +1,122 @@
+#--------------------------------------------------------------------------------
+#
+# Test Steps:
+#--------------------------------------------------------------------------------
+# - Deploy RHCS 5 cluster in RHEL 9
+# - Check the MSGRV2 parameters
+# - Upgrade cluster from rhcs 5 to rhcs 6
+# - Check the MSGRV2 paramters and Mclock parameters
+#--------------------------------------------------------------------------------
+#################################################################################
+tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                rhcs-version: 5.3
+                release: "latest"
+                mon-ip: node1
+                orphan-initial-daemons: true
+                skip-monitoring-stack: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: bootstrap and deployment services with label placements.
+      polarion-id: CEPH-83573777
+      destroy-cluster: false
+      module: test_cephadm.py
+      name: Deploy RHCS cluster using cephadm
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        node: node7
+        install_packages:
+          - ceph-common
+        copy_admin_keyring: true
+        caps:                             # authorize client capabilities
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
+      desc: Configure client node on the Cluster
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+  - test:
+      name: Ceph paramter checking in 5.x
+      desc: MSGRV2 parameter checking in 5.x
+      polarion-id: CEPH-83574890
+      module: test_config_parameter_chk.py
+      config:
+        scenario : msgrv2_5x
+  - test:
+      name: Upgrade cluster to latest 6.x ceph version
+      desc: Upgrade cluster to latest version
+      module: test_cephadm_upgrade.py
+      polarion-id: CEPH-83573791,CEPH-83573790
```

Comment:
CEPH-83573791 -> Test marked inactive in polarion

Reply:
Please check this is an active test case.
Polorain link - https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573791
I copied the upgrade part from the "suites/quincy/upgrades/tier-1_upgrade_5x-to-6x.yaml"